packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -30,7 +32,7 @@ BuildRequires: %{?scl_prefix_ruby}rubygems
 BuildRequires: %{?scl_prefix}rubygem(deface) < 2.0
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0
-BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 2.0.2
+BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 2.0
```

Comment:
I think we can actually really drop this. It's now only a dev dependency so the tests can run together. https://github.com/theforeman/foreman-packaging/pull/3120 might still be good to finish though.